src/app/beer_garden/db/mongo/api.py

```diff
@@ -75,7 +75,13 @@ def from_brewtils(obj: ModelItem) -> MongoModel:
         The Mongo model item
 
     """
-    model_dict = SchemaParser.serialize(obj, to_string=False)
+    if isinstance(obj, brewtils.models.Garden):
```

Comment:
This makes me nervous.  I feel like one of the benefits of having the schema live in beer-garden should be to also get away from the dependence on the brewtils models.  That's a thing that we should do by moving away from this whole `to_brewtils` and `from_brewtils` dance over time where we can.

This looks like it's here so that existing code can attempt to use these new schemas as a sort of drop-in replacement where they had been using the brewtils ones.  I haven't gone dumpster diving into all of the places where that might be the case, but if that's what is indeed happening here my gut tells me that this is going to lead to subtly different and difficult to debug behavior.